### ISSUE TITLE
Android - Steps Screen - Current Step Not Denoted Visually for Sub-navigation

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#36]
Why when on a “Topics” screen does it not tell which the user what “Step” they are on? I would expect since you have what I'm calling a sub-navigation that the current screen should be highlighted.

The user is on Step 1, so the "1" should be made to standout from the other steps in the sub-navigation.

Please see attached screenshot of sub-navigation. 

![android - steps screen - step 1 with sub-navigation not highlighted](https://user-images.githubusercontent.com/10423487/57238541-a78aaf80-6ff7-11e9-8dd3-7cbe53fd8703.png)

**Comment by KyleLivingstonDOL** [comment]
Hi John,

I believe this has been fixed and will be demo'ed during the next 508 compliance check.

Kyle

**Comment by jfc3-dol** [comment]
When does sprint 7 end and when can I see the application, give or take a day or two?

**Comment by KyleLivingstonDOL** [comment]
UAT beings June 10.

**Comment by jfc3-dol** [comment]
The Step screens I randomly checked now have a half of a gold circle to denote which step a person is on.

So I'm going to close this issue.
